### ISSUE TITLE
Expose response object

### ISSUE THREAD

**Issue by HaNdTriX** [brendonboshell/supercrawler#15]
First of all thanks a lot for this nifty crawler. I am really enjoying your api design!

Nevertheless, would it be possible to expose the entire [response object](https://github.com/brendonboshell/supercrawler/blob/master/lib/Crawler.js#L236-L241) in the handler api? Currently supercrawler only exposes `contentType`, `response.body` and `url`.

This would allow us to crawl a lot [more information](https://nodejs.org/api/http.html#http_class_http_incomingmessage): 

* crawl response headers
* handle http status codes/messages
* performing custom redirects

I can create a PR if such a change would be acceptable.

**Comment by brendonboshell** [comment]
Yes, I would accept a pull request for this provided the change was backwards compatible. Adding a `response` property to the [`ctx object`](https://github.com/brendonboshell/supercrawler/blob/master/lib/Crawler.js#L320) seems the most obvious route?
